catalog/spatial/geocoding/spatial-geocoding-plugin/src/main/java/org/codice/ddf/spatial/geocoding/plugin/GeoCoderPlugin.java

```diff
@@ -118,11 +106,31 @@ public int getRadius() {
         return radiusInKm;
     }
 
-    private boolean countryCodeNeeded(Metacard metacard) {
-        return metacard.getAttribute(Location.COUNTRY_CODE) == null;
+    private void setCountryCodeAttributeIfNeeded(Metacard metacard) {
```

Comment:
Not to be nit-picky... but I do not love the names of these functions.

setCountryCode or updateCountryCode instead of setCountryCodeIfNeeded reads a lot better, a simple explanation in a method documentation would make it more readable.

Also, I think hasCountryCode would be a lot better than countryCodeNeeded